_core/_proto/board_config_template.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
This is the template file, test.php would json_encode this and write it to a config.json, which imgboard would then load and parse.